Src/contrib/library/web/authentication/oauth/cypress/consumer/src/data/oauth_jwt_config.e

```diff
@@ -0,0 +1,53 @@
+note
+	description: "Summary description for {OAUTH_JWT_CONFIG}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	OAUTH_JWT_CONFIG
+
+inherit
+
+	OAUTH_CONFIG
+		rename
+			make as make_config
```

Comment:
I find suspicious that we inherit from OAUTH_CONFIG ... and then we barely use the ancestor ...

maybe in `make`... call `			make_default (a_api_key, "")`